src/components/Layout/HomeContent.js

```diff
@@ -233,14 +233,15 @@ export function HomeContent() {
         <Section background="right-card">
           <Center>
             <Header>
-              Go full-stack <br className="hidden lg:inline" />
-              with a framework
+              Full-stack <br className="hidden lg:inline" />
+              com um framework
             </Header>
             <Para>
-              React is a library. It lets you put components together, but it
-              doesn’t prescribe how to do routing and data fetching. To build an
-              entire app with React, we recommend a full-stack React framework
-              like <Link href="https://nextjs.org">Next.js</Link> or{' '}
+              React é uma biblioteca. Ele permite que você junte componentes,
+              mas não prescreve como fazer roteamento e busca de dados. Para
+              construir um aplicativo inteiro com React, recomendamos um
+              framework React full-stack como{' '}
```

Comment:
```suggestion
              framework React full stack como{' '}
```

Vejo o termo full stack ser usado no português sem hífen em todas as situações. Acredito que é o mais correto.